ptsprojects/zephyr/gatt.py

```diff
@@ -64,6 +64,29 @@ def __get_attr_u16_hdl_uc_str(offset):
     return '{0:04x}'.format(iut_attr_db_off + offset, 'X')
 
 
+def __add_unique_char_to_service(i=[0]):
```

Comment:
That's a nice hack but in general, we want to move away from global variables, so let's use a Stack object here to store the service UUID. This way it will be more readable and testable.

Reply:
Switched to `stack.gatt` object